### PR TITLE
Introduces ProcedureExecution - closes #109

### LINKED ISSUE #109: Missing a class in SSN that generalize Observation/Actuation/Sampling
In SSN there is ssn:Property that generalizes `sosa:ObservableProperty` and `sosa:ActuatableProperty`.
There is `ssn:System` that generalize `sosa:Sensor`, `sosa:Actuator`, `sosa:Sampler`.

It would be appropriate to have a class `ssn:ProcedureExecution` that generalizes `sosa:Observation`, `sosa:Actuation`, `sosa:Sampling`

### PR DESCRIPTION
**Pull request by maximelefrancois86**
https://raw.githack.com/w3c/sdw-sosa-ssn/procedure-execution/ssn/index.html#SSNProcedureExecution

### CODE REVIEW

File: ssn/index.html [hunk 1]
```diff
@@ -2557,6 +2577,31 @@ <h5><a href="#SSNOutput">ssn:Output</a></h5>
               <span class="back">[<a href="#overview-of-classes-and-properties">Back to top</a>]</span>
             </div>
         </section>
+
+        <section class="specterm ssn" id="SSNProcedureExecution" about="ssn:ProcedureExecution" typeof="owl:Class rdfs:Class">
```
Comment (dr-shorthair) [inline ssn/index.html]: sosa: ns?

**Review by dr-shorthair — CHANGES_REQUESTED** [review]
in spec doco change to `sosa:ProcedureExecution`

**Comment by maximelefrancois86** [comment]
something is messing up line endings. I'll figure that out.

File: ssn/index.html [hunk 2]
```diff
@@ -2557,6 +2577,31 @@ <h5><a href="#SSNOutput">ssn:Output</a></h5>
               <span class="back">[<a href="#overview-of-classes-and-properties">Back to top</a>]</span>
             </div>
         </section>
+
+        <section class="specterm ssn" id="SOSAProcedureExecution" about="sosa:ProcedureExecution" typeof="owl:Class rdfs:Class">
```
Comment (dr-shorthair) [inline ssn/index.html]: Is `specterm ssn` correct here?

File: ssn/index.html (hunk 1 above)
Comment (dr-shorthair) [reply ssn/index.html]: Hmm. Still showing ssn:

File: ssn/index.html (hunk 2 above)
Comment (maximelefrancois86) [reply ssn/index.html]: it's not. well spotted. corrected in commit [c25c0bc](https://github.com/w3c/sdw-sosa-ssn/pull/130/commits/c25c0bc99de362159255bad9c1d3f0ea6efe5184)

**Review by dr-shorthair — APPROVED** [review]

**Review by dr-shorthair — APPROVED** [review]

**Review by nicholascar — APPROVED** [review]

**Pull request merged**